src/main/kotlin/io/github/yuutoproject/yuutobot/commands/base/CommandCategory.kt

```diff
@@ -18,12 +18,14 @@
 
 package io.github.yuutoproject.yuutobot.commands.base
 
-enum class CommandCategory {
-    INFO,
-    FUN,
-    UTIL,
-    OTHER_CATEGORY;
+enum class CommandCategory(val emote: String) {
+    INFO("\u2139"),
+    FUN("\uD83C\uDFB2"),
+    UTILITIES("\u2699"),
+    OTHER("\uD83D\uDCC2");
```

Comment:
Is this other category used?

Reply:
It's not, I didn't remove it earlier because I assumed it was a necessary category.

Reply:
Nah that was there for nothing, it's safe to remove it